Source/Core/Gen3/Searchers/StaticSearcher3.cpp

```diff
@@ -67,10 +67,18 @@ void StaticSearcher3::startSearch(const std::array<u8, 6> &min, const std::array
 }
 
 std::vector<SearcherState> StaticSearcher3::search(u8 hp, u8 atk, u8 def, u8 spa, u8 spd, u8 spe,
-                                                   const StaticTemplate *staticTemplate) const
+                                                   const StaticTemplate3 *staticTemplate) const
 {
     std::vector<SearcherState> states;
-    std::array<u8, 6> ivs = { hp, atk, def, spa, spd, spe };
+    std::array<u8, 6> ivs;
+    if (staticTemplate->getBuggedRoamer()) {
+        atk &= 7;
+        ivs = { hp, atk, 0, 0, 0, 0 };
```

Comment:
ivs = { hp, atk & 7, 0, 0, 0, 0 };

Reply:
I get this error:
![image](https://github.com/Admiral-Fish/PokeFinder/assets/20956021/e80be9cb-c5bb-4fd4-bd1d-5095ca6b2a4c)

`StaticSearcher3.cpp:76:21: Non-constant-expression cannot be narrowed from type 'int' to 'unsigned char' in initializer list (fix available)`

Reply:
static_cast<u8>(atk & 7)

Reply:
Done